src/OVAL/public/oval_definitions.h

```diff
@@ -251,6 +251,7 @@ typedef enum {
 oval_family_t oval_subtype_get_family(oval_subtype_t);
 const char *oval_operator_get_text(oval_operator_t);
 const char *oval_subtype_get_text(oval_subtype_t);
+const char *oval_subtype_to_str(oval_subtype_t);
```

Comment:
Let's mark this as deprecated and replace all occurrences with the get_text variant.